addon/services/portal-service.js

```diff
@@ -256,4 +256,9 @@ export default Service.extend(serviceMixin, {
     return this._post(urlPath, opts, portalOpts);
   },
 
+  getUserDefaultSettings (portalOpts) {
```

Comment:
Is this something that should ultimately be pushed down into rest-js?

Reply:
That's an interesting question as I'm not sure this is a "published api". That said, it would be great to move the rest of the `user-service` calls to rest-js